_posts/2022-12-20-Ecco-l-anteprima-del-nuovo-Designers-Italia

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+lang: it
+excerpt_separator: <!-- MORE -->
+title: "Ecco l’anteprima del nuovo Designers Italia"
+subtitle: "Esploralo all’indirizzo dedicato e costruiscilo insieme a noi"
+date: 2022-12-20 09:00:00 +0200
+pinned: false
+nodate: false
+homeBottomLink: false
+homeHideCategories: true
+date_prepend: "Articolo"
+categories:
+  - cittadini
+  - pubblica amministrazione
+---
+
+<!-- MORE -->
+**Una Pubblica Amministrazione più vicina alle persone, semplice e utile: [prossima.designers.italia.it](/)** non è solo **la prossima iterazione** del sito di Designers Italia, il progetto del Dipartimento per la trasformazione digitale e di Agenzia per l’Italia digitale (AGID), ma soprattutto un’ulteriore evoluzione dell’iniziativa che **crea e diffonde strumenti pratici** per progettare e realizzare servizi pubblici digitali di qualità e **cura una community** che si confronta sulle buone pratiche di settore.
```

Comment:
you need https:// at the beginning of the url... @danielaiozzo

Reply:
ops you need the full url: https://prossima.designers.italia.it on the right between ()

Reply:
the text between [] is ok instead

Reply:
done